Src/AutoFakeItEasy.FakeItEasy2.UnitTest/AutoFakeItEasy.FakeItEasy2.UnitTest.csproj

```diff
@@ -0,0 +1,103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If we want to ensure that AutoFakeItEasy works with FakeItEasy 1.x and 2.y, then I'm not sure how.
I suppose there could be some test runner trickery where the same project is run twice, by swapping in FakeItEasy 2.x for the second test run (and adding the redirect). That was not something I'd felt up to proposing, as I'm unsure of the mechanism and am not confident that I could introduce such a thing into the build system. However, if it appeals, I'd try.

I doubt there'd be much maintenance for this test project. We could replace the `Compile Include`s with a more sophisticated mechanism [as used in xBehave](https://github.com/xbehave/xbehave.net/blob/ed11fcdda746edb9a111ac1b109d4d37f13a7c2a/tests/Xbehave.Test.Acceptance.Net45/Xbehave.Test.Acceptance.Net45.csproj#L80-L84), if you think it's likely that new files will be added to the original test project and you worry about them being forgotten here.